README.rst

```diff
@@ -96,13 +96,14 @@ Quick Example
     from nextcord.ext import commands
 
 
-    bot = commands.Bot(command_prefix='$')
+    bot = commands.Bot()
 
-    @bot.command()
-    async def ping(ctx):
-        await ctx.reply('Pong!')
 
-    bot.run('token')
+    @bot.slash_command(description="Replies with pong!")
+    async def ping(interaction: nextcord.Interaction):
+        await interaction.send("Pong!", ephemeral=True)
+        
```

Comment:
there seems to be trailing whitespace on this line, could you remove the unnecessar spacing like shown in pre-commit?

Reply:
Check now please.

Reply:
I'm being serious, what are you doing? Just have a look at the diff here, you see you added 2 extra not-needed newlines, remove those, not anything else.

Reply:
There you go.

Reply:
You'll want to read my last sentence and the failing check.

Reply:
Pre-commit should have fixed it by now. Sorry, didn't understand what you meant clearly.